tests/components/utils-test.js

```diff
@@ -237,6 +237,30 @@ describe('Utils', function () {
     expect(isValidString).toBeTruthy();
   });
 
+  it('tests sanitizeConfigData', function () {
+    var data = null;
+    var sanitizedConfigData = Utils.sanitizeConfigData(data);
+    expect(sanitizedConfigData).toEqual({});
+
+    data = '';
```

Comment:
Strings and numbers are valid, the data type we need to prevent here are Arrays.

The deepmerge function already checks if value is a "[mergeable Object](https://github.com/KyleAMathews/deepmerge/blob/master/index.js#L11)"